src/main/resources/hudson/plugins/jobConfigHistory/deleteRevisionAndTableEntry.js

```diff
@@ -0,0 +1,51 @@
+//name is an optional parameter for job deletion history and system config history
+function removeEntryFromTable(id, timestamp, name, message) {
+    var confirmPhrase = message + timestamp + '?';
+    if (confirm(confirmPhrase)) {
+        var tableRow = document.getElementById(id);
+        tableRow.parentNode.removeChild(tableRow);
+
+        //redirect
+        var xmlHttp = new XMLHttpRequest();
+        var url = 'deleteRevision?timestamp=' + timestamp;
+        if (name != null) {
+            url += "&name=" + name;
+        }
+        xmlHttp.open("POST", url, true);
+        xmlHttp.setRequestHeader(document.head.getAttribute('data-crumb-header'), document.head.getAttribute('data-crumb-value'));
+        xmlHttp.send(null);
+    } else {
+        return false;
+    }
+}
+
+document.addEventListener('DOMContentLoaded', function() {
+
+    const systemConfigDeleteButtons = document.querySelectorAll('.jenkins-button--destructive');
+    const agentConfigDeleteButtons = document.querySelectorAll('.delete-button');
```

Comment:
Declaration is too far away from its usage, which makes the code harder to read. See e.g. https://checkstyle.sourceforge.io/checks/coding/variabledeclarationusagedistance.html